sql/expression/function/format_test.go

```diff
@@ -33,50 +33,161 @@ func TestFormat(t *testing.T) {
+		{"blob is ok", sql.Blob, sql.Int32, sql.NewRow([]byte{1, 2, 3}, nil, nil), nil, nil},
+		{"text int without d", sql.Text, sql.Int32, sql.NewRow("98765432", nil, nil), nil, nil},
+		{"text int with d", sql.Text, sql.Int32, sql.NewRow("98765432", 2, nil), "98,765,432.00", nil},
+		{"text int with negative d", sql.Text, sql.Int32, sql.NewRow("98765432", -1, nil), "98,765,432", nil},
+		{"text int with float d", sql.Text, sql.Float64, sql.NewRow("98765432", 2.123, nil), "98,765,432.00", nil},
+		{"text int with float negative d", sql.Text, sql.Float64, sql.NewRow("98765432", float32(-1), nil), "98,765,432", nil},
+		{"text float without d", sql.Text, sql.Int32, sql.NewRow("98765432.1234", nil, nil), nil, nil},
+		{"text float with d", sql.Text, sql.Int32, sql.NewRow("98765432.1234", 2, nil), "98,765,432.12", nil},
+		{"text float with negative d", sql.Text, sql.Int32, sql.NewRow("98765432.8234", -1, nil), "98,765,433", nil},
+		{"text float with float d", sql.Text, sql.Float64, sql.NewRow("98765432.1234", float64(2.823), nil), "98,765,432.123", nil},
+		{"text float with float negative d", sql.Text, sql.Float64, sql.NewRow("98765432.1234", float64(-1), nil), "98,765,432", nil},
+		{"text float with blob d", sql.Text, sql.Blob, sql.NewRow("98765432.1234", []byte{1, 2, 3}, nil), nil, nil},
+		{"negative num text int with d", sql.Text, sql.Int32, sql.NewRow("-98765432", 2, nil), "-98,765,432.00", nil},
+		{"sci-notn big num", sql.Float64, sql.Int32, sql.NewRow(5932886+.000000000001, 1, nil), "5,932,886.0", nil},
+		{"sci-notn big num with big dp", sql.Float64, sql.Int32, sql.NewRow(5932886+.000000000001, 8, nil), "5,932,886.00000000", nil},
+		{"sci-notn big exp num", sql.Float64, sql.Int32, sql.NewRow(5.932887e+08, 2, nil), "593,288,700.00", nil},
+		{"sci-notn neg big exp num", sql.Float64, sql.Int32, sql.NewRow(-5.932887e+08, 2, nil), "-593,288,700.00", nil},
+		{"sci-notn text big exp num", sql.Text, sql.Int32, sql.NewRow("5.932887e+07", 3, nil), "59,328,870.000", nil},
+		{"sci-notn text neg big exp num", sql.Text, sql.Int32, sql.NewRow("-5.932887e+08", 2, nil), "-593,288,700.00", nil},
+		{"sci-notn exp small num", sql.Float64, sql.Int32, sql.NewRow(5.932887e-08, 2, nil), "0.00", nil},
+		{"sci-notn exp small num with big dp", sql.Float64, sql.Int32, sql.NewRow(5.932887e-08, 9, nil), "0.000000059", nil},
+		{"sci-notn neg exp small num", sql.Float64, sql.Int32, sql.NewRow(-5.932887e-08, 2, nil), "0.00", nil},
+		{"sci-notn neg exp small num with big dp", sql.Float64, sql.Int32, sql.NewRow(-5.932887e-08, 8, nil), "-0.00000006", nil},
+		{"sci-notn text neg exp small num", sql.Float64, sql.Int32, sql.NewRow("-5.932887e-08", 2, nil), "0.00", nil},
+		{"sci-notn text neg exp small num with big dp", sql.Float64, sql.Int32, sql.NewRow("-5.932887e-08", 8, nil), "-0.00000006", nil},
+		//{"sci-notn big num with big dp", sql.Float64, sql.Int32, sql.NewRow(5932886+.000000000001, 15, nil), "5,932,886.000000000001000", nil},
```

Comment:
Rather than commenting these out, create a new test method TestSkippedFormat() just like this one to hold them. Then in each `t.Run()`, call `t.Skip()`

Doing it that way makes the skips visible in the results to anyone running the tests so we don't just forget about them.

Reply:
Add a comment above your new test method about what kind of differences there are for the skipped locales